public/index.html

```diff
@@ -24,10 +21,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Shards Dashboard Lite React</title>
```

Comment:
Looks like it's still not an MFD related title in this PR, you may run into some merge conflicts that you will have to resolve after you merge one of these PRs. We probably don't need the base files (App.js, index.html etc) to be changed in every PR but it's OK as first PR's. 
If you do run into merge conflicts in these PRs just make sure to select the right version (MFD title).